repl/appengine/web/src/app/reference_panel/reference-panel-component.html

```diff
@@ -16,8 +16,46 @@
 
 <h1>References</h1>
 
-<p>See <a href="https://github.com/google/cel-spec">github.com/google/cel-spec</a> for CEL language
-    overview. </p>
-<p>See
-    <a href="https://github.com/google/cel-go/tree/master/repl/main">github.com/google/cel-go/tree/master/repl/main</a>
-    for REPL syntax guide.</p>
\ No newline at end of file
+<mat-accordion>
+    <mat-expansion-panel>
```

Comment:
Expansion panels unfortunately won't push the height of its parent container when expanded. So you end up seeing a cut-off content with a scrollbar when the container isn't tall enough to display the expanded content.

You could either change the primary container to something that supports fullscreen (mat-sidenav-container) or just set the viewport heights in your container to be full (not standard angular practice, but it works):

```
.content {
    display: block;
    margin-top: 5vh;
    width: 100%;
    height: 95vh;
}
```

Reply:
Done: I had to restructure the top level app a bit but should behave more like a full screen page.